x-pack/plugins/apm/server/routes/services/get_service_overview_container_metadata.ts

```diff
@@ -67,8 +61,10 @@ export const getServiceOverviewContainerMetadata = async ({
     { exists: { field: KUBERNETES_DEPLOYMENT_NAME } },
   ];
 
-  const response = await esClient.search<unknown, ResponseAggregations>({
-    index: [indexName],
+  const response = await infraMetricsClient.search<
+    unknown,
+    ResponseAggregations
+  >({
```

Comment:
This shouldn't be needed
```suggestion
  const response = await infraMetricsClient.search({
```